<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>550</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblCanvas">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>801</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="action_Save"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
    <widget class="QMenu" name="menuPen">
     <property name="title">
      <string>Pen</string>
     </property>
     <addaction name="action_PenRed"/>
     <addaction name="action_PenGreen"/>
     <addaction name="action_PenBlue"/>
    </widget>
    <widget class="QMenu" name="menuTrans">
     <property name="title">
      <string>Trans</string>
     </property>
     <addaction name="action_Grayscale"/>
    </widget>
    <addaction name="menuPen"/>
    <addaction name="menuTrans"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>about</string>
    </property>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTool"/>
   <addaction name="menuInfo"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_PenRed"/>
   <addaction name="action_PenGreen"/>
   <addaction name="action_PenBlue"/>
   <addaction name="separator"/>
   <addaction name="action_About"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/new.png</normaloff>:/icon/imgs/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/open.png</normaloff>:/icon/imgs/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/save.png</normaloff>:/icon/imgs/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/exit.png</normaloff>:/icon/imgs/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action_PenRed">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/red.png</normaloff>:/icon/imgs/red.png</iconset>
   </property>
   <property name="text">
    <string>Red(R)</string>
   </property>
  </action>
  <action name="action_PenGreen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/green.png</normaloff>:/icon/imgs/green.png</iconset>
   </property>
   <property name="text">
    <string>Green(G)</string>
   </property>
  </action>
  <action name="action_PenBlue">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/blue.png</normaloff>:/icon/imgs/blue.png</iconset>
   </property>
   <property name="text">
    <string>Blue(B)</string>
   </property>
  </action>
  <action name="action_About">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/imgs/about.png</normaloff>:/icon/imgs/about.png</iconset>
   </property>
   <property name="text">
    <string>System Info</string>
   </property>
  </action>
  <action name="action_Grayscale">
   <property name="text">
    <string>GrayScale</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
